server_admin/topics/roles-groups/proc-creating-realm-roles.adoc

```diff
@@ -2,7 +2,12 @@
 
 = Creating a realm role
 [role="_abstract"]
+ifeval::[{project_community}==true]
+Realm-level roles are a namespace for defining your roles. To see the list of roles, click *Realm Roles* in the menu.
+endif::[]
+ifeval::[{project_product}==true]
 Realm-level roles are a namespace for defining your roles. To see the list of roles, click *Roles* in the menu.
+endif::[]
 
 image:{project_images}/roles.png[]
 
```

Comment:
Step one in the procedure should be

ifeval::[{project_product}==true]
. Click *Add Role*.
endif::[]
ifeval::[{project_project}==true]
. Click *Create Role*.
endif::[]